keys.schema.json

```diff
@@ -126,6 +129,163 @@
         }
       },
       "required": ["packageId", "csrftoken", "sessionid"]
+    },
+    "Safari": {
+      "type": "object",
+      "properties": {
+        "zip": {
+          "type": "string"
+        },
+        "bundleId": {
+          "type": "string",
+          "description": "i.e. com.plasmo.mock"
+        },
+        "extensionBundleId": {
+          "type": "string",
+          "description": "i.e. com.plasmo.mock.extension"
+        },
+        "appName": {
+          "type": "string",
+          "description": "i.e. Plasmo Mock"
+        },
+        "appCategory": {
+          "type": "string",
+          "description": "last component of LSApplicationCategoryType (i.e. business)"
+        },
+        "platforms": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "defaults to iOS and macOS"
+        },
+        "buildNumber": {
+          "type": "number",
+          "description": "defaults to GITHUB_RUN_NUMBER"
+        },
+        "appleId": {
```

Comment:
Are all these emails required or they are optional, and if any of them exist, the internal use it? - Quite intriguing, might be a bit overwhelm to the dev that uses this? I wonder if some of these email are just duplicate of each other? (Tho I assume they need to be separated for fastlane?)